code/modules/hydroponics/hydroponics.dm

```diff
@@ -876,6 +876,17 @@
 
 /obj/machinery/hydroponics/proc/harvest_plant(mob/user)
 	if(harvest)
+		if(HAS_TRAIT(user, FOOD_JOB_BOTANIST))
+			var/random_increase = rand(1, 20) * 0.01
+			if(prob(50))
+				myseed.adjust_potency(round(myseed.potency *random_increase, 1), FALSE)
+			if(prob(20))
+				myseed.adjust_yield(round(myseed.yield * random_increase, 1), FALSE)
+			if(prob(35))
+				myseed.adjust_lifespan(round(myseed.lifespan * random_increase, 1), FALSE)
+			if(prob(40))
+				myseed.adjust_endurance(round(myseed.endurance * random_increase, 1), FALSE)
```

Comment:
Numbers may need to be balanced over time, but will benefit longer rounds for sure.